src/set_theory/ordinal/notation.lean

```diff
@@ -451,10 +451,14 @@ def mul : onote → onote → onote
   if e₂ = 0 then oadd e₁ (n₁ * n₂) a₁ else
   oadd (e₁ + e₂) n₂ (mul o₁ a₂)
 
+-- Better unfolding.
```

Comment:
What does this comment mean?

Reply:
For whatever reason, deleting this instance (even though it's redundant) causes some breakage later down. I believe the problem is that there's more things to unfold, which breaks some simps. That's what I was going for with this comment.

Reply:
I removed this instance and fixed the places where this broke.